packages/valory/skills/trader_abci/handlers.py

```diff
@@ -64,6 +81,15 @@
 
 
 PREDICT_AGENT_PROFILE_PATH = "predict-ui-build"
+CHATUI_PARAM_STORE = "chatui_param_store.json"
+
+AVAILABLE_TRADING_STRATEGIES = frozenset(
+    {
+        "kelly_criterion_no_conf",
+        "bet_amount_per_threshold",
+        "mike_strat",
+    }
+)
```

Comment:
You can reuse the existing enum instead.

```python
from packages.valory.skills.trader_abci.prompts import TradingStrategy
```

```suggestion
AVAILABLE_TRADING_STRATEGIES = frozenset(strategy.value for strategy in TradingStrategy)
```